scripts/install-tree-sitter-lib

```diff
@@ -16,4 +14,5 @@ fi
   git clean -dfX
   make
   sudo make install
+  sudo ldconfig /usr/local/lib
```

Comment:
This could be restricting when installing semgrep from source to users.

Reply:
We don't need to install the tree-sitter lib in `/usr/local/lib` (if that's what you mean). I could add a `--prefix` option to our script and call tree-sitter's configure script.